index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>reviewtrial project</title>
+  </head>
+  <body>
+    <!-- survey section  -->
+    <section class="survey-section">
+      <h1 class="title">Ismail Survey Form</h1>
```

Comment:
- [ ] Nice work implementing the inline comments as a reviewer. It would be nice to be always polite when conducting reviews rather than dishing out orders. For example, this comment can be put in this way _**"It would be nice if your our title in line 7 and the h1 tag title in line 12 are the same for the following reasons....**_